tests/unit_gcs/test_dataproc.py

```diff
@@ -0,0 +1,63 @@
+import uuid
+from skyplane.config_paths import cloud_config
+from skyplane.obj_store.object_store_interface import ObjectStoreInterface
+from tests.interface_util import interface_test_framework
+from skyplane.utils import imports
+from google.cloud import dataproc_v1 as dataproc
+from google.cloud import compute_v1 as compute
+
+def test_dataproc():
+    
+    
+    cluster_name = f"skyplane-dataproc-test-acf"
+    region = "us-central1"
+    project_id ="skyplane-hailey-373919"
```

Comment:
Do you know how to retrieve the  GCP project ID of the running skyplane project? I hard-coded mine because I was running it under mine, but I wasn't able to find how I can retrieve the project ID from the running pipeline.